Minio.Functional.Tests/FunctionalTest.cs

```diff
@@ -1323,8 +1323,8 @@ private static void runQuickTests(MinioClient minioClient)
                 string bucketName = GetRandomName(15);
                 await Setup_Test(minio, bucketName);
 
-                await ListObjects_Test(minio, bucketName, null, 0);
-
+                ListObjects_Test(minio, bucketName, null, 0);
```

Comment:
@nitisht , line 1326 is missing a Wait(). You could do  ListObjects_Test(minio, bucketName, null, 0).Wait(5000) instead to achieve the same as line 1326 and 1327

Reply:
ah, sure will do @poornas

Reply:
done